.github/ISSUE_TEMPLATE/create_release_branch.md

```diff
@@ -0,0 +1,157 @@
+---
+name: "[Runbook] Create release branch"
+about: Create a new branch for a new stable Kubernetes release
+---
+
+#### Summary
+
+Make sure to follow the steps below and ensure all actions are completed and signed-off by one team member.
+
+#### Information
+
+<!-- Replace with the version to create the branch for, e.g. 1.28 -->
+- **K8s version**: 1.xx
+
+<!-- Set this to the name of the person responsible for running the release tasks, e.g. @neoaggelos -->
+- **Owner**:
+
+<!-- Set this to the name of the team-member that will sign-off the tasks -->
+- **Reviewer**:
+
+<!-- Link to PR to initialize the release branch (see below) -->
+- **PR**:
+
+#### Actions
+
+The steps are to be followed in-order, each task must be completed by the person specified in **bold**. Do not perform any steps unless all previous ones have been signed-off. The **Reviewer** closes the issue once all steps are complete.
+
+- [ ] **Owner**: Add the assignee and reviewer as assignees to the GitHub issue
+- [ ] **Owner**: Ensure that you are part of the ["containers" team](https://launchpad.net/~containers)
+- [ ] **Owner**: Request a new `1.xx` Snapstore track for the snaps similar to the  [snapstore track-request][].
+  - #### Post template on https://discourse.charmhub.io/
+
+    **Title:** Request for 1.xx tracks for the k8s snap
+
+    **Category:** store-requests
+
+    **Body:**
+
+      Hi,
+
+      Could we please have a track "1.xx-classic" and "1.xx" for the respective K8s snap release?
+
+      Thank you, $name
+
+- [ ] **Owner**: Create `release-1.xx-strict` branch from latest `autoupdate/strict`
+  - `git switch autoupdate/strict`
+  - `git pull`
+  - `git checkout -b release-1.xx-strict`
+  - `git push origin release-1.xx-strict`
+- [ ] **Owner**: Create `release-1.xx` branch from latest `main`
+  - `git switch main`
+  - `git pull`
+  - `git checkout -b release-1.xx`
+  - `git push origin release-1.xx`
+- [ ] **Owner**: Create `1.xx` branch from latest `master` in k8s-dqlite
+  - `git clone git@github.com:canonical/k8s-dqlite.git ~/tmp/release-1.xx`
+  - `pushd ~/tmp/release-1.xx`
+  - `git switch main`
+  - `git pull`
+  - `git checkout -b 1.xx`
+  - `git push origin 1.xx`
+  - `popd`
+  - `rm -rf ~/tmp/release-1.xx`
+- [ ] **Owner**: Create `release-1.xx` branch from latest `main` in cilium-rocks
```

Comment:
Are all these steps for creating release branches in multiple places something we could do in a script?

Reply:
VERY LIKELY this gets automated... the only reason it's not in this doc is to be clear of the tasks